Mage/src/main/java/mage/abilities/Mode.java

```diff
@@ -119,4 +121,17 @@ public Mode withCost(Cost cost) {
     public Cost getCost() {
         return cost;
     }
+
+    public void setPawPrintValue(int pawPrintValue) {
```

Comment:
what's the point of having separate "set" and "with" methods? I would just have the "with" method, the return can be ignored if not needed